docker/rootfs/replace_config.py

```diff
@@ -22,6 +25,8 @@
 uwsgi_filedata = uwsgi_filedata.replace("uwsgihost", os.environ.get("UWSGI_HOST"))
 uwsgi_filedata = uwsgi_filedata.replace("uwsgiprotocol", os.environ.get("UWSGI_PROTOCOL"))
 
-
-with open('/opt/prom2teams/uwsgi.ini', 'w') as file:
-  file.write(uwsgi_filedata)
+try:
+  with open('/opt/prom2teams/uwsgi.ini', 'w') as file:
+    file.write(uwsgi_filedata)
+except (PermissionError, OSError) as error:
```

Comment:
same same here

Reply:
Change to tmp config files in case of config file from docker volume.